node/service/src/chain_spec.rs

```diff
@@ -39,72 +39,81 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
-			balances: endowed_accounts.iter().map(|&k|(k, 1u64 << 60)).collect(),
+			balances: endowed_accounts.iter().map(|&k|(k, 10_000_000 * DOLLARS)).collect(),
+			transaction_base_fee: 1 * CENTS,
+			transaction_byte_fee: 10 * MILLICENTS,
+			existential_deposit: 1 * DOLLARS,
+			transfer_fee: 1 * CENTS,
+			creation_fee: 1 * CENTS,
+			reclaim_rebate: 1 * CENTS,
 		}),
 		session: Some(SessionConfig {
 			validators: initial_authorities.iter().cloned().map(Into::into).collect(),
-			session_length: 60,	// that's 5 minutes per session.
+			session_length: 5 * MINUTES
 		}),
 		staking: Some(StakingConfig {
 			current_era: 0,
 			intentions: initial_authorities.iter().cloned().map(Into::into).collect(),
-			offline_slash: Perbill::from_millionths(1000),
-			session_reward: Perbill::from_billionths(60),
+			offline_slash: Perbill::from_billionths(1_000_000),
+			session_reward: Perbill::from_billionths(2_065),
 			current_offline_slash: 0,
 			current_session_reward: 0,
-			validator_count: 12,
-			sessions_per_era: 12,	// 1 hour per era
-			bonding_duration: 24 * 60 * 12,	// 1 day per bond.
+			validator_count: 7,
+			sessions_per_era: 12,
+			bonding_duration: 1 * DAYS,
 			offline_slash_grace: 4,
 			minimum_validator_count: 4,
 		}),
 		democracy: Some(DemocracyConfig {
-			launch_period: 12 * 60 * 24,	// 1 day per public referendum
-			voting_period: 12 * 60 * 24 * 3,	// 3 days to discuss & vote on an active referendum
-			minimum_deposit: 5000,	// 12000 as the minimum deposit for a referendum
+			launch_period: 5 * MINUTES,	// 1 day per public referendum
+			voting_period: 5 * MINUTES,	// 3 days to discuss & vote on an active referendum
+			minimum_deposit: 50 * DOLLARS,	// 12000 as the minimum deposit for a referendum
 		}),
 		council: Some(CouncilConfig {
 			active_council: vec![],
-			candidacy_bond: 5000,	// 5000 to become a council candidate
-			voter_bond: 1000,		// 1000 down to vote for a candidate
-			present_slash_per_voter: 1,	// slash by 1 per voter for an invalid presentation.
-			carry_count: 6,		// carry over the 6 runners-up to the next council election
-			presentation_duration: 12 * 60 * 24,	// one day for presenting winners.
-			approval_voting_period: 12 * 60 * 24 * 2,	// two days period between possible council elections.
-			term_duration: 12 * 60 * 24 * 24,	// 24 day term duration for the council.
-			desired_seats: 0, // start with no council: we'll raise this once the stake has been dispersed a bit.
-			inactive_grace_period: 1,	// one addition vote should go by before an inactive voter can be reaped.
-
-			cooloff_period: 12 * 60 * 24 * 4, // 4 day cooling off period if council member vetoes a proposal.
-			voting_period: 12 * 60 * 24, // 1 day voting period for council members.
+			candidacy_bond: 10 * DOLLARS,
+			voter_bond: 1 * DOLLARS,
+			present_slash_per_voter: 1 * CENTS,
+			carry_count: 6,
+			presentation_duration: 1 * DAYS,
+			approval_voting_period: 2 * DAYS,
+			term_duration: 28 * DAYS,
```

Comment:
Is that intentionally that `term_duration` changes from 24 to 28 days?

Reply:
yeah